init/init.go

```diff
@@ -78,10 +78,10 @@ func Boot() {
 
 // Listen for systemctl requests
 func Serve() {
-	switch conf.Method {
+	switch Conf.Method {
 	case "http":
-		if err := listenHTTP(conf.Port.String()); err != nil {
-			log.Fatalf("Error starting server on %s: %s", conf.Port, err)
+		if err := listenHTTP(Conf.Port.String()); err != nil {
+			log.Fatalf("Error starting server on %s: %s", Conf.Port, err)
 		}
```

Comment:
probably worth adding a warning here for the default case, in case people make a typo or omit the config file and are wondering why `systemctl` does nothing.